src/app/inventory/InventoryItem.tsx

```diff
@@ -53,7 +54,10 @@ export default class InventoryItem extends React.Component<Props> {
     const badgeInfo = getBadgeInfo(item);
 
     const elaborateTile =
-      $featureFlags.forsakenTiles && item.isDestiny2() && (item.primStat || item.sockets);
+      $featureFlags.forsakenTiles &&
+      settings.betaForsakenTiles &&
```

Comment:
Unfortunately in the React world you should not use settings directly. You have to fish it out from the Redux store. Otherwise it won’t update.